src/mrdb_frontend/src/components/LoginPage.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <!DOCTYPE html>
+  <html lang="en">
+      <head>
+        <link href="https://fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet">
+        <meta charset="UTF-8">
+        <title>MRDB Application</title>
+      </head>
+  </html>
+  <body>
+    <div class="main">
+        <p class="sign" align="center">Sign in</p>
+        <form class="form1">
+            <input id="userName" class="un" type="text" align="center" placeholder="Username">
+            <input id="password" class="pass" type="password" align="center" placeholder="Password">
+            <a class="submit" align="center" v-on:click="validate">Sign in</a>
+            <p class="forgot" align="center"><a href="#">Forgot Password?</a></p>
+            <p class="forgot" align="center"><a href="/register">New User? Register Here</a></p>
+        </form>
+    </div>
+  </body>
+</template>
+
+<script>
+export default {
+  name: 'LoginPage',
+  data() {
+    return {
+      msg: ''
+    }
+  },
+  methods: {
+    async validate() {
+        const data = {userName: document.getElementById('userName').value,
+                      password: document.getElementById('password').value};
+        const requestData = {
+                    method: "POST",
+                    headers: {"Content-Type": "application/json"},
+                    body: JSON.stringify(data)
+        };
+        const response = await fetch("/api/users/login", requestData);
```

Comment:
Why is the `login` process happening in the validate method? The name validate indicates that the implementation just does validation and nothing else.